src/components/menu/tests/position.spec.ts

```diff
@@ -0,0 +1,308 @@
+// (C) 2007-2018 GoodData Corporation
```

Comment:
please move the file to +` <tested tode dir>/tests`

Reply:
Moved.